### DISCUSSION TITLE
Capture and store stdout/stderr in testcase output directory

### DISCUSSION THREAD

**Discussion by nicsav** [pytest-dev/pytest#10742] [General]
Is there a way to redirect stdout/stderr for each test individually. That is to have something like stdout.log/stderr.log in the testcase's output directory. Or some other named for the combined stdout/stderr capture.

I find that helpful for post-mortem debugging as well as for easily sharing testcase runs with others.

I tried --capture and --show-capture but they don't seem to allow that.

**Comment by RonnyPfannschmidt** [comment]
Currently there is no capture mode that safes to files
